desktop/apps/consign/client/actions.js

```diff
@@ -321,6 +321,8 @@ export function ignoreRedirectOnAuth () {
 export function logIn (values, accountCreated = false) {
   return async (dispatch, getState) => {
     try {
+      dispatch(startLoading())
```

Comment:
Is `startLoading` related to a loader of some kind? Do you think it might be clearer if it were `showLoader` and `hideLoader`?

Reply:
Ah! I don't have a huge opinion on this-- the boolean field is `loading`, so `startLoading` and `stopLoading` make sense in that context, and it's generic because it refers to a single loading state on each page (where anything could be happening). Could definitely change but I don't mind this either. /shrug

Reply:
Yah, I think `show` and `hide` is better unless there's an actual request taking place within those actions; "startLoading" makes me immediately think that theres a big process happening behind the scenes.